### ISSUE TITLE
[Bug][A11y][Switch] Read-only variant is not accessible

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#521]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

iPhone14+

### OS Version

iOS26

### App version

0.6.0 (520)

### Describe the issue

Read-only switch has no role/trait so it's not reached with screen-reader or other assistive tools.

### Expected Behavior

Read-only switch should have accessible role/trait, label and value(on/off), like an active switch but without the possibility to change the value.
You can use the "value" or "hint" property to convey the read-only state
